organizers/views.py

```diff
@@ -250,10 +250,13 @@ def get_application(self, kwargs):
         :return: pending aplication that has not been voted by the current
         user and that has less votes and its older
         """
+        max_votes_per_app = getattr(settings, 'MAX_VOTES_PER_APP', 50)
         return models.Application.objects \
             .exclude(vote__user_id=self.request.user.id) \
+            .exclude(user_id=self.request.user.id) \
```

Comment:
pots posar tot això en una sola linia:
```suggestion
            .exclude(vote__user_id=self.request.user.id, user_id=self.request.user.id) \
```